### DISCUSSION TITLE
What's meaning of Conn Busy?

### DISCUSSION THREAD

**Discussion by marcopolooo** [jackc/pgx#2056] [Q&A]
Hello guys, I hope you are in good condition.

I would like to ask, about "conn busy" I get the error when I have a website that hit my API more than 1 one endpoint, for example, one page the website hits/categories, /report at the same time. Then I got the error "conn busy".

How do I fix that situation? Thank you very much

**Comment by jackc** [comment]
`*pgx.Conn` can only do one thing at a time. `conn busy` means there is a bug in your application or in pgx itself where an attempt is made to use a connection that is already in use.

Based on your description, I think the problem is you are using `*pgx.Conn` instead of `*pgxpool.Pool`.
